<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>方程工具</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,5">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,1,2">
     <item>
      <widget class="QLineEdit" name="fangcheng1">
       <property name="placeholderText">
        <string>请输入一元方程（x为未知数，乘方用**，系数与x用*）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>= 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button1">
       <property name="text">
        <string>解一元方程</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="fangcheng2">
       <property name="placeholderText">
        <string>请输入方程组，未知数为x1,x2,x3...                                        至多解十元方程，每个方程格式为ax1+bx2+cx3...-d，代表ax1+bx2+cx3...-d，注意不要写=0                                   方程与方程之间要换行</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="button2">
         <property name="text">
          <string>解多元方程</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button3">
         <property name="text">
          <string>扩展</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
